.github/workflows/permissions.yml

```
@@ -0,0 +1,10 @@
+name: permissions
+on: pull_request
```

If you validate only this file - do you think it makes sense to run the action only on when it changes?

```suggestion
on: pull_request
    paths:
      - install/installer/pkg/components/spicedb/data/schema.yaml
```

---

Sure, makes sense. Is there an easy way to achieve that? Or do I need to dust-off my bash?

---

It's the suggestion in my comment 😄 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

---

Ah, sorry! I thought it was just how the diff highlighted!